### PR TITLE
docs: update wasm public docs

### PR DESCRIPTION
**Pull request by kristifidani**
# Pull Request Template

## Motivation and Context

Explain why this change is necessary or the issue it addresses.

## Summary

-  Update the public docs
- The technical docs is moved towards an internal readme file which is not published.

## Checklist

Please ensure that your PR meets the following requirements:

### General

- [ ] Code follows project style and guidelines.
- [ ] No redundant or duplicate code.
- [ ] No added new dependencies without clear justification and approval.

### Documentation

- [ ] Added/updated relevant documentation (e.g., README, inline comments).
- [ ] Added a CHANGELOG entry for major changes.

### Testing

- [ ] Added relevant test cases, leveraging tools like `rstest` or similar for reusable tests.
- [ ] Tests pass locally.

### Build & CI/CD

- [ ] Confirmed no CI/CD pipeline issues.
- [ ] Updated build scripts where necessary.

## Comments to Reviewer

Please provide any comments or points of consideration for the reviewers, such as specific areas of focus, potential edge cases, or context that would aid the review process.

### Reviewer Responsibilities

- [ ] Code adheres to style and guidelines.
- [ ] All tests are appropriately covered and they pass.
- [ ] Consider performance and/or security impacts.
- [ ] Constructive feedback is provided.

### CODE REVIEW

File: sdk/bindings/wasm/public-README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# ETOPay SDK – WASM Bindings
+
+This package provides ***WebAssembly (WASM)*** bindings for the ETOPay SDK, enabling seamless integration with both _Web_ applications and _NodeJS_ environments. It generates TypeScript type definitions for all relevant data objects, ensuring type safety and ease of use. Additionally, this package includes functionality for publishing both the WASM binary and the generated bindings to an NPM registry.
```
Comment (antbern) [inline sdk/bindings/wasm/public-README.md:3]: do we need to mention the part about publishing bindings to NPM?
```suggestion
This package provides ***WebAssembly (WASM)*** bindings for the ETOPay SDK, enabling seamless integration with both _Web_ applications and _NodeJS_ environments. It generates TypeScript type definitions for all relevant data objects, ensuring type safety and ease of use.
```

File: sdk/bindings/wasm/public-README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+### 🚀 Quickstart
+
+For a hands-on example of how to integrate the ETOPay SDK, check out our public quickstart guide:
+👉 [ETOPay SDK Quickstart (TypeScript)](https://github.com/ETOSPHERES-Labs/etopay-sdk-quickstart-ts).
+
+### 📖 Usage
+
+Before integrating the ETOPay SDK, you must create a project via our [ETOPay Dashboard](https://etopayapp.etospheres.com). Once registered, you will receive some `credentials`, which are required for setting up and using the SDK.
```
Comment (antbern) [inline sdk/bindings/wasm/public-README.md]:
```suggestion
Before integrating the ETOPay SDK, you must create a project via our [ETOPay Dashboard](https://etopayapp.etospheres.com). Once registered, you will receive some `credentials` and the configuration, which are required for setting up and using the SDK.
```

File: sdk/bindings/wasm/public-README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+This package provides ***WebAssembly (WASM)*** bindings for the ETOPay SDK, enabling seamless integration with both _Web_ applications and _NodeJS_ environments. It generates TypeScript type definitions for all relevant data objects, ensuring type safety and ease of use. Additionally, this package includes functionality for publishing both the WASM binary and the generated bindings to an NPM registry.
+
+### 🚀 Quickstart
+
+For a hands-on example of how to integrate the ETOPay SDK, check out our public quickstart guide:
+👉 [ETOPay SDK Quickstart (TypeScript)](https://github.com/ETOSPHERES-Labs/etopay-sdk-quickstart-ts).
+
+### 📖 Usage
+
+Before integrating the ETOPay SDK, you must create a project via our [ETOPay Dashboard](https://etopayapp.etospheres.com). Once registered, you will receive some _credentials_, which are required for setting up and using the SDK.
```
Comment (kristifidani) [inline sdk/bindings/wasm/public-README.md:12]:
```suggestion
Before integrating the ETOPay SDK, you must create a project via our [ETOPay Dashboard](https://etopayapp.etospheres.com). Once registered, you will receive some `credentials` and the configuration, which are required for setting up and using the SDK.
```

File: sdk/bindings/wasm/public-README.md (hunk 2 above)
Comment (kristifidani) [reply sdk/bindings/wasm/public-README.md]: Added another comment to apply the suggestion for this 👍

**Review by antbern — APPROVED** [review]

**Review by hauju — APPROVED** [review]

**Review by antbern — APPROVED** [review]
